modules/form-field/react/stories/stories_TextArea.tsx

```diff
@@ -3,14 +3,69 @@ import * as React from 'react';
 import {storiesOf} from '@storybook/react';
 import withReadme from 'storybook-readme/with-readme';
 import {StaticStates} from '@workday/canvas-kit-labs-react-core';
-import {controlComponent, ComponentStatesTable, permutateProps} from '../../../../utils/storybook';
+import {
+  controlComponent,
+  ComponentStatesTable,
+  permutateProps,
+  customColorTheme,
+} from '../../../../utils/storybook';
 
 import {TextArea} from '../../../text-area/react';
 import FormField from '../index';
 import README from '../../../text-area/react/README.md';
 
 const hintText = 'Helpful text goes here.';
 const hintId = 'error-desc-id';
+const TextAreaStates = () => (
```

Comment:
Nit: can you move this to the bottom of the file above the visual testing stories? That way it's not the first thing people see when they look for examples.